File: app/src/main/java/com/example/nasaenterpriseapi/Model/Items.java
package com.example.nasaenterpriseapi.Model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Items {

	@SerializedName("data")
	private List<Data> data;

	@SerializedName("href")
	private String href;

	@SerializedName("links")
	private List<Links> links;

	public List<Data> getData(){
		return data;
	}

	public String getHref(){
		return href;
	}

	public List<Links> getLinks(){
		return links;
	}
}

File: app/src/main/java/com/example/nasaenterpriseapi/model/NasaImages/Items.kt
package com.example.nasaenterpriseapi.model.NasaImages

import com.google.gson.annotations.SerializedName

data class Items (

	@SerializedName("links") val links : List<Links>,
	@SerializedName("href") val href : String,
	@SerializedName("data") val data : List<Data>
)